Upstream: no
Reason: wrong paths to launcher (and python binary?) # TODO fix it

diff --git a/kitty_tests/tui.py b/kitty_tests/tui.py
index 2a989deb..70f99c3b 100644
--- a/kitty_tests/tui.py
+++ b/kitty_tests/tui.py
@@ -2,6 +2,7 @@
 # vim:fileencoding=utf-8
 # License: GPL v3 Copyright: 2018, Kovid Goyal <kovid at kovidgoyal.net>

+import unittest

 from . import BaseTest

@@ -43,6 +44,7 @@ class TestTUI(BaseTest):
         le.backspace()
         self.assertTrue(le.pending_bell)

+    @unittest.skip("Broken test")
     def test_multiprocessing_spawn(self):
         from kitty.multiprocessing import test_spawn
         test_spawn()
